randovania/games/dread/json_data/Artaria.txt

```diff
@@ -4827,6 +4815,10 @@ Extra - asset_id: collision_camera_081
   * Extra - actor_def: actordef:actors/props/totalrechargestation/charclasses/totalrechargestation.bmsad
   * Extra - start_point_actor_name: PRP_TotalR_Platform_000
   * Extra - start_point_actor_def: actordef:actors/props/weightactivatedplatform_total/charclasses/weightactivatedplatform_total.bmsad
+  > Door to Transport to Burenia
+      All of the following:
+          Before Artaria - Screw Attack Rotatable
+          Simple IBJ or Use Spin Boost
```

Comment:
This connection needs removed. From below, you can only access the door after the rotatable has been used, from Under Left Side Blocks.